solutions/core-landing-zone/securitycontrols.md

```diff
@@ -438,7 +438,9 @@
 |IA-1|./org/org-policies/iam-disable-service-account-key-upload.yaml|iam-disable-service-account-key-upload|
 |SC-20|./lz-folder/services-infrastructure/dns-project/dns.yaml|dns-project-id-standard-core-public-dns|
 |SC-20|./lz-folder/services-infrastructure/dns-project/dns.yaml|dns-project-id-standard-core-public-dns|
-|SI-4|./lz-folder/audits/logging-project/cloud-logging-buckets.yaml|platform-and-component-log-bucket|
+
```

Comment:
It seems like the next three lines are out of place.

Reply:
Something strange here:

![image](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/assets/13170368/93edecd9-5605-42b1-b0ce-d7a9fbd31607)

Reply:
right, anoop can you fix this ?

Reply:
done thanks